test/scenarios/iframe/iframe.crank.yml

```diff
@@ -0,0 +1,13 @@
+scenario: IFrame focus test
+description: Tests focusability of an iframe
+  
+steps:
+- step: Navigate to http://www.automatoninc.com/hubspot-test/
+- waitFor: 5
+  step: Click the page element [id=cookie_action_close_header]
+- step: focus on the iframe[id=hs-form-iframe-0] frame
+- step: Fill out input[name=firstname] with Test
+- step: Fill out input[name=lastname] with IFrame
+- step: Fill out input[name=email] with frametest@automatoninc.com
+- step: submit the form by clicking input[type=submit]
+- step: Then the email field on hubspot contact frametest@automatoninc.com should be frametest@automatoninc.com
```

Comment:
- [ ] Nit: recommend adding a `Finally, delete the frametest@automatoninc.com hubspot contact` just to help prevent false positives in the future.